### ISSUE TITLE
more resources for generating factor graph

### ISSUE THREAD

**Issue by MohammadrezaDindarloo** [symforce-org/symforce#362]
hello everyone, I wanted to create my custom factor to add it to nonlinear factor graph from gtsam. 
I searched I found one example in tutorial but I need some more example to understanding the concept. I will be greatfull if anyone have any example to help me in this mean.

thank you 
m.r.dindarloo

**Comment by aaron-skydio** [comment]
Not sure which example you're referring to that you've already found, but here's some of what we have in the docs:

- The homepage has examples of some more symbolic residual functions, if that's the part you're looking for help with: https://symforce.org/index.html#build-an-optimization-problem
- It also shows how to generate a factor into C++: https://symforce.org/index.html#generate-runtime-c-code
- The "custom factor generation" example also shows how to define a symbolic factor and generate it into C++: https://symforce.org/examples/custom_factor_generation/README.html

What part of defining, generating, and using a factor are you looking for help with?

**Comment by MohammadrezaDindarloo** [comment]
hello, 
I checked the docs and it was helpful to me. the main part that I was dealing with was in generating a custom factor in order  using in GTSAM factors. At first I write my model with sf. and then tried to create factor that I couldn't find a good resource to do it, finally I ended up with making jacobian with symforce library and add them manually in my factor graph in gtsam and it worked. Now I will try to do this again in symforce by generating my factor from an specific model with define variables and add the output factor to my nonlinearfactorgraph gtsam. 
I should have more study to find the details more because I am new in factor graph and using symforce to generate my factor so if any simple example is developed it will be great to me, but in this level I found this library very powerful in creating the model jacobian (before that I used maple and I coudn't generate the output).

thank you.
